assets/_scss/elements/_typography.scss

```diff
@@ -17,12 +17,22 @@ p {
 }
 
 a {
-  color: $color-base-link;
-  &:hover, &:active, &:visited {
-    color: $color-base-link;
+  border-bottom: 1px dotted;
+  &:hover {
+    border-bottom: 1px solid;
   }
 }
 
+.usa-link {
```

Comment:
Classes should eventually be moved out of elements. This is partly due to organizational reasons: you would only expected to see bare html elements in an elements folder. This will also help with a trending upward specificity curve when you import the files. So elements imported first (lowest specificity), then components (medium specificity) then patterns (high specificity) etc.